rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -293,19 +297,29 @@ public static ABICallElection deserializeElection(byte[] data, AddressBasedAutho
         return new ABICallElection(authorizer, votes);
     }
 
-    public static byte[] serializeLockWhitelist(LockWhitelist whitelist) {
-        List<Address> whitelistAddresses = whitelist.getAddresses();
-        int serializationSize = whitelistAddresses.size() * 2 + 1;
+    public static byte[] serializeOneOffLockWhitelist(LockWhitelist whitelist) {
+        List<LockWhitelistEntry> whitelistEntries = whitelist.getEntries().stream().filter(entry -> entry instanceof OneOffWhiteListEntry).collect(Collectors.toList());
```

Comment:
Probably compare using `getClass()` rather than `instanceof` given that we don't care about inheritance.